vuepress/docs/next/tutorials/solution/install-standard-demo.md

```diff
@@ -62,120 +89,142 @@ ent ecr deploy --repo="https://github.com/entando-samples/standard-demo-manage-u
+     :::
 
-You can now navigate to your application's home page using the home icon in the upper right of the App Builder.
+## Access the Standard Banking Demo
 
+Access the Standard Banking Demo via one of the following options.
 
-**Option 2** Alternatively, you can view the Standard Banking Demo home page by going to `Pages → Management`, finding `Home SD` in the page tree, and clicking `View Published Page` from its actions.
+### Option 1
 
+Make the Standard Banking Demo your default home page. The home icon shortcut will then navigate to the Standard Banking Demo home page. 
+
+1. Go to `App Builder` → `Pages` → `Settings` 
+2. In the dropdown for Home Page, select `Home / Home SD`
+3. Click `Save`
+![HomepageSelect.png](./images/HomepageSelect.png)
+4. Click the home icon in the upper right of the App Builder
+
+
+### Option 2 
+
+Navigate to the Standard Banking Demo home page from your App Builder page tree.
+
+1. Go to `App Builder` → `Pages` → `Management` 
+2. Find `Home SD` in the page tree
+3. From the `Actions` pull-down menu, go to `View Published Page`
 ![Standard Banking Demo Homepage](./images/Homepage.png)
 
 
 ## Application Details
 
-The Entando Standard Banking Demo application demonstrates a number of the major features of the Entando platform, including:
+The Entando Standard Banking Demo application demonstrates a number of the major features of the Entando Platform:
 
 * Keycloak integration for role based access controls
-* Micro frontends implemented using React and Angular and co-existing on the same dashboard page
+* React and Angular micro frontends that coexist on the same dashboard
 * Micro frontend communication techniques
-* Microservices via Spring Boot
+* Spring Boot Microservices
 * Entando Content Management
 
-### Micro Frontends (MFE)
+### Micro Frontends
 
-The application includes six MFEs in which the above features complement one another to achieve custom functionality. These are described below.
+The application includes six micro frontends (MFEs) showcasing complementary features and custom functionality.
 
-#### 1. Card
+#### 1. Card MFE
 
 ![SeedCard.png](./images/SeedCard.png)
 
-- The Card MFE is a React micro frontend that is visible on the My Dashboard page. The MFE makes an API call to the banking microservice to fetch a numeric result depending on the configured card type. The displayed value will change as the configuration is changed.
-- The MFE is authorization-aware and will pass the bearer token to the microservice for authorization and authentication. If you render the dashboard page and you aren't authenticated, the widget displays an error message.
-- This MFE emits events that are consumed by the Transaction Table widget.
+- A React micro frontend placed on the My Dashboard page.
+- Makes an API call to the banking microservice to fetch a numeric result dependent on the current card selection. The displayed value changes with changes to the card configuration.
```

Comment:
first sentence: Slightly confusing & awkward since the first sentence does not have a subject but has a period to make it a sentence>> unsure how to structure these details about the MFEs consistently. though after a while, i got used to the format but the first time reading the list, this one threw me off.

Second sentence: The displayed value changes with updates to the card configuration.